bundles/org.openhab.binding.electroluxair/README.md

```diff
@@ -0,0 +1,111 @@
+# ElectroluxAir Binding
+
+This is an openHAB binding for the Pure A9 Air Purifier, by Electrolux.
+
+This binding uses the Electrolux Delta REST API.
+
+![Electrolux Pure A9](doc/electrolux_pure_a9.webp)
+
+## Supported Things
+
+This binding supports the following types:
+
+- Bridge
+- Electrolux Pure A9
+
+If you define the bridge in a things-file the bridge type id is defined as `bridge`, e.g.:
+
+`Bridge electroluxair:api:myAPI "Electrolux Delta API" [username="user@password.com", password="12345", refresh="300"`
+
+If you define the thing in a things-file the thing type id is defined as:
+
+`Thing electroluxpurea9 MyElectroluxPureA9  "Electrolux Pure A9"    [ deviceId="123456789" ]`
+
+## Discovery
+
+After the configuration of the Bridge, your Electrolux Pure A9 device will be automatically discovered and placed as a thing in the inbox.
+
+
+### Configuration Options
+
+#### Bridge
+
+| Parameter | Description                                                  | Type   | Default  | Required | 
+|-----------|--------------------------------------------------------------|--------|----------|----------|
+| username  | The username used to connect to the Electrolux Wellbeing app | String | NA       | yes      |
+| password  | The password used to connect to the Electrolux Wellbeing app | String | NA       | yes      |
+| refresh   | Specifies the refresh interval in second                     | Number | 600      | yes      |
+
+## Thing Configuration
+
+Only the bridge require manual configuration. The thing can be added by hand, or you can let the discovery mechanism automatically find your Electrolux Pure A9 thing.
+
+### Configuration Options
```

Comment:
This heading is already present a few lines above.

Reply:
Fixed!